### DISCUSSION TITLE
Make it possible to export maps to android

### DISCUSSION THREAD

**Discussion by durshlakk** [ppy/osu#28155] [Ideas]
Hi! I am a person who does not have access to a computer for the next 5 months and has a desire to make maps for me, this function is very necessary to upload maps. Make it active :)
![Screenshot_20240512-060116](https://github.com/ppy/osu/assets/77969719/7fd9edc0-117c-4e8c-bbab-64b3a9420936)

**Comment by BlockHead007** [comment]
[#27869](https://github.com/ppy/osu/discussions/27869#discussioncomment-9115509)
